editor/plugins/org.fusesource.ide.wsdl2rest.ui/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.core.databinding,
  org.jboss.tools.foundation.core,
  org.jboss.tools.foundation.ui,
- org.fusesource.ide.foundation.core,
+ org.fusesource.ide.foundation.core;visibility:=reexport,
```

Comment:
better to always avoid to "reexport", if another plugin needs the dependencies, it should declare the dependencies in its MANIFEST too